generators/app/templates/index.html

```diff
@@ -6,13 +6,13 @@
                 "baseUrl": "view/frontend/web/js/"
             };
         </script>
-        <script data-main="App" src="https://unpkg.co/requirejs@2.3.3/require.js"></script>
+        <script data-main="App" src="https://unpkg.com/requirejs@2.3.3/require.js"></script>
         <script src="view/frontend/requirejs-config.js"></script>
         <script>
             (function (require) {
                 Object.assign(config.paths, {
-                    'react': '//unpkg.com/react/dist/react',
-                    'react-dom': '//unpkg.com/react-dom/dist/react-dom',
+                    'react': '//unpkg.com/react@16.0.0/umd/react.production.min',
```

Comment:
Upgrading to React 16 might have some breaking changes for our applications, we need to be careful with this. If we merge this, we're going to have to go through and make sure everything still works as expected & refactor where necessary.

We do need to upgrade at some point but do we have time right now?

https://reactjs.org/blog/2017/09/26/react-v16.0.html

Reply:
Want me to drop back to 15.4 for now?